src/components/DigitalCredentials/CredentialsDashboard.vue

```diff
@@ -0,0 +1,153 @@
+      attach="#app"
+      @close="credentialRevokedDialog = false"
+    />
+
+    <RevokeCredentialErrorDialog
+      :dialog="revokeCredentialErrorDialog"
+      attach="#app"
+      @close="revokeCredentialErrorDialog = false"
+    />
+
+    <v-container>
+      <v-row>
+        <v-col
+          cols="12"
+          md="8"
+        >
+          <CredentialsTable
+            v-if="issuedCredentials?.length"
+            :issuedCredentials="issuedCredentials"
+            @onPromptConfirmRevokeCredential="displayConfirmRevokeCredentialDialog"
+            @onPromptConfirmReplaceCredential="displayConfirmReplaceCredentialDialog"
+          />
+          <CredentialsLanding v-else />
+        </v-col>
+        <v-col
+          cols="12"
+          md="4"
+        >
+          <CredentialsInfo />
+        </v-col>
+      </v-row>
+    </v-container>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { useBusinessStore } from '@/stores'
+import { Getter } from 'pinia-class'
+import { DigitalCredentialIF } from '@/interfaces'
+import { LegalServices } from '@/services'
+import CredentialsInfo from '@/components/DigitalCredentials/CredentialsInfo.vue'
+import CredentialsLanding from '@/components/DigitalCredentials/CredentialsLanding.vue'
+import CredentialsTable from '@/components/DigitalCredentials/CredentialsTable.vue'
+import ConfirmReplaceCredentialDialog from '@/components/DigitalCredentials/dialogs/ConfirmReplaceCredentialDialog.vue'
+import ConfirmRevokeCredentialDialog from '@/components/DigitalCredentials/dialogs/ConfirmRevokeCredentialDialog.vue'
+import CredentialRevokedDialog from '@/components/DigitalCredentials/dialogs/CredentialRevokedDialog.vue'
+import ReplaceCredentialErrorDialog from '@/components/DigitalCredentials/dialogs/ReplaceCredentialErrorDialog.vue'
+import RevokeCredentialErrorDialog from '@/components/DigitalCredentials/dialogs/RevokeCredentialErrorDialog.vue'
+import { Routes } from '@/enums'
+
+@Component({
+  components: {
+    CredentialsInfo,
+    CredentialsLanding,
+    CredentialsTable,
+    ConfirmRevokeCredentialDialog,
+    ConfirmReplaceCredentialDialog,
+    CredentialRevokedDialog,
+    ReplaceCredentialErrorDialog,
+    RevokeCredentialErrorDialog
+  }
+})
+export default class CredentialsDashboard extends Vue {
+  @Getter(useBusinessStore) getIdentifier!: string;
+
+  confirmRevokeCredentialDialog = false;
+  confirmReplaceCredentialDialog = false;
+  credentialRevokedDialog = false;
+  replaceCredentialErrorDialog = false;
+  revokeCredentialErrorDialog = false;
+  issuedCredential: DigitalCredentialIF = null;
+  issuedCredentials: Array<DigitalCredentialIF> = [];
+
+  async mounted (): Promise<void> {
+    await this.getCredentials()
+  }
+
+  async getCredentials (): Promise<void> {
+    const { data } = await LegalServices.fetchCredentials(this.getIdentifier)
+    this.issuedCredentials = data?.issuedCredentials
```

Comment:
Conditional chaining may result in an undefined value, which isn't reactive in Vue (and you use this variable as a prop) so I recommend assigning a fallback value (`|| []`).

Reply:
Looks good.